.pre-commit-config.yaml

```
@@ -9,7 +9,7 @@ repos:
         args: ["--py37-plus"]
 
   - repo: https://github.com/PyCQA/flake8
-    rev: 6.1.0
+    rev: 7.0.0
```

```suggestion
    rev: 6.1.0
```
This was already done in a separate pull request. ;-)